src/.vscode/settings.json

```diff
@@ -4,5 +4,6 @@
     "**/bower_components": true,
     "**/*.code-search": true,
     "**/PublicAPI*.txt": true
-  }
+  },
+  "editor.rulers": [100]
```

Comment:
Added this so by default you get a vertical line in VS Code at 100 characters. Not sure if this is something you want, in that case I'll drop the commit.